Notes and Outlines/Outlines/gh-pages_outline.md

```diff
@@ -0,0 +1,43 @@
+# First Draft at Defining the GH Pages Course
+
+* What is GitHub Pages?
+  - Conceptual Overview
+    + A place to host web pages for free.
+    + Jekyll lets you do this w/o writing HTML/CSS. Includes blogging.
+  - Potential Use Cases
+    + Personal blog.
+    + Portfolio site.
+    + Intro to GH projects.
+    + etc.
```

Comment:
Not sure if this matters for this audience but there's lots of interest in using Pages sites for documentation. :wink: